src/sage/combinat/designs/incidence_structures.py

```diff
@@ -1810,8 +1810,7 @@ def dual(self, algorithm=None):
 
         REFERENCE:
 
-        - Soicher, Leonard, Design package manual, available at
-          https://www.gap-system.org/Manuals/pkg/design/htm/CHAP003.htm
+        - Soicher, Leonard, :gap_package:`Design package manual <design/htm/CHAP003.htm>`
         """
```

Comment:
I got a request from someone to replace "Soicher, Leonard" with "Leonard Soicher". Please.

Reply:
OK, let me press Cmd-G and make this change

Reply:
Thanks. What do you mean by "pressing Cmd-G"??

Reply:
Is it an emacs command?

Reply:
Perhaps searching next item?

Reply:
Ah, I got it: "interrupting current commands" :-)

Reply:
Here on GitHub, if you press Cmd-G in a comment, it creates a "suggestion"

Reply:
Ah I see. Thanks.